<!DOCTYPE html>
<html lang="en">

<head>
	<meta charset="utf-8"/>
	<title>New website openings</title>
	<meta name="viewport" content="width=device-width, initial-scale=1.0">

	<!-- RSS feed -->
	<link rel="alternate" type="application/rss+xml" title="Data science lab" href="http://datasciencelabs.org/feed.xml">

  <!-- Customized Bootstrap + Font Awesome + Solarized -->
  <link href="/css/style.css" rel="stylesheet" media="screen">

	<!-- Favicon -->
	<link rel="shortcut icon" href="/images/favicon.png"/>

	<!-- Typekit -->
	<script>
	  (function(d) {
		var config = {
		  kitId: 'xeu8jut',
		  scriptTimeout: 3000
		},
		h=d.documentElement,t=setTimeout(function(){h.className=h.className.replace(/\bwf-loading\b/g,"")+" wf-inactive";},config.scriptTimeout),tk=d.createElement("script"),f=false,s=d.getElementsByTagName("script")[0],a;h.className+=" wf-loading";tk.src='//use.typekit.net/'+config.kitId+'.js';tk.async=true;tk.onload=tk.onreadystatechange=function(){a=this.readyState;if(f||a&&a!="complete"&&a!="loaded")return;f=true;clearTimeout(t);try{Typekit.load(config)}catch(e){}};s.parentNode.insertBefore(tk,s)
	  })(document);
	</script>

	<!-- Google Analytics -->
<!--	<script>-->
<!--		(function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){-->
<!--		(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),-->
<!--		m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)-->
<!--		})(window,document,'script','//www.google-analytics.com/analytics.js','ga');-->

<!--		ga('create', 'UA-26244371-2', 'bedford.io');-->
<!--		ga('send', 'pageview');-->

<!--	</script>-->

	<!-- jQuery -->
	<script src="/js/jquery.min.js"></script>

	<!-- Bootstrap -->
	<script src="/js/bootstrap.min.js"></script>

</head>

<body>

	<div id="header">
		<nav class="navbar navbar-expand-md navbar-light bg-navbar">
			<div class="container">
				
				<a class="navbar-brand" href="/">
					<img class="logo navbar-brand d-inline-block align-top ml-0 mr-1" src="/images/logo.svg">
					data science lab
				</a>
				
				<button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarNav" aria-controls="navbarNav" aria-expanded="false" aria-label="Toggle navigation">
					<span class="navbar-toggler-icon"></span>
				</button>
				<div class="collapse navbar-collapse" id="navbarNav">
					<ul class="navbar-nav nav-pills ml-auto">
						
						<li class="nav-link active">
						
						<a class="mx-1" href="/blog/">blog</a>
						</li>
						
						<li class="nav-link">
						
						<a class="mx-1" href="/papers/">papers</a>
						</li>
						
						<li class="nav-link">
						
						<a class="mx-1" href="/projects/">projects</a></li>
						
						<li class="nav-link">
						
						<a class="mx-1" href="/team/">team</a></li>
					</ul>
				</div>
			</div>
		</nav>
	</div>

	<div class="container mt-4">

	<div class="blog-post">
	<div class="row">
		<div class="col-lg-1"></div>
		<div class="col-lg-10">
			<div class="media">
				<div class="media-body">
					<div class="title media-heading">
						<a href="/blog/great-news-about-prizes/" class="off">New website openings</a>
					</div>
					<p>
					<div class="smallhead">
						30 Sep 2022
						
						by <a class="off" href="
							
								
							
								
							
								
							
								
							
								
							
								
							
								
							
								
							
								
							
								
							
								
							
								
							
								
							
								
							
								
							
								
									/team/sejin-chun/
								
							
								
							
						">
						Sejin Chun
						</a>
						
						
					</div>
				</div>
			</div>
		</div>
		<div class="col-lg-1"></div>
	</div>

	<div class="bigspacer"></div>

	<div class="row">
		<div class="col-lg-1"></div>
		<div class="col-lg-10">
			
			<div class="post">
				<p>Positions for a <strong>bioinformatician</strong> and a <strong>full-stack developer</strong> are available immediately in the Bedford lab at the Fred Hutch. Details for both positions follow:</p>

<hr />

<h3 id="bioinformatician">Bioinformatician</h3>

<p>We have an opening for a bioinformatician in the Bedford lab at the Fred Hutch to work on genomic epidemiology and evolutionary analysis of pathogens including SARS-CoV-2, influenza and Ebola virus. This position will contribute to ongoing work on two major projects: Nextstrain and Seattle Flu Study.</p>

<p><a href="https://nextstrain.org">Nextstrain</a> is an award-winning tool for tracking infectious disease epidemics developed in collaboration with the <a href="https://neherlab.org">Neher lab</a> at the University of Basel. Nextstrain won the <a href="https://www.nih.gov/news-events/news-releases/open-science-prize-announces-epidemic-tracking-tool-grand-prize-winner">Open Science Prize</a> in Feb 2017 and has been instrumental in analysis of the <a href="https://nextstrain.org/sars-cov-2">SARS-CoV-2 pandemic</a>, <a href="https://nextstrain.org/community/inrb-drc/ebola-nord-kivu">Ebola outbreaks</a>, <a href="https://nextstrain.org/zika/">Zika spread</a> in the Americas and is used by the World Health Organization to aid in the process of <a href="https://nextstrain.org/flu/">influenza vaccine strain selection</a>.</p>

<p>The <a href="https://seattleflu.org/">Seattle Flu Study</a> is a collaboration of groups at the Brotman Baty Institute, the Fred Hutch, the University of Washington, and Seattle Children’s. Already in its third year, this study has produced high-resolution analyses of the spread of SARS-CoV-2 and influenza in Seattle by building a software platform that processes subject and sample metadata, lab assay results, and raw and processed genome data in near-real time.</p>

<h4 id="responsibilities">Responsibilities</h4>

<p>The role involves both development and maintenance of bioinformatic analyses and pipelines which underpin both projects’ research aims. This will involve a mixture of independent work, collaboration with scientists in the group and interactions with the wider community. The vast majority of code is open-source. Specific examples from Nextstrain include analytic pipelines that clean and ingest genome metadata, construct consensus genomes, and build phylogenetic trees, as well as tools to enable a diverse range of collaborators to run SARS-CoV-2 analyses through Nextstrain. Work on Seattle Flu Study focuses on pipelines to assemble raw sequence data into consensus SARS-CoV-2 and influenza genomes and deposition of these consensus genomes to public databases.</p>

<p>Interfacing with project collaborators in-person and online is a key aspect of this position. The bioinformatician will work within a small team of existing members of the Bedford lab and the larger research group of the Seattle Flu Study. The Nextstrain team communicates openly about project and organizational decisions and encourages participation by all team members in the decision-making process.</p>

<h4 id="qualifications">Qualifications</h4>

<h5 id="minimum-qualifications">Minimum qualifications</h5>

<ul>
  <li>Fluency in at least one high-level programming language, such as Python, R, Ruby, JavaScript or Perl</li>
  <li>Knowledge of molecular biology</li>
  <li>Motivated to learn new skills and technologies</li>
  <li>Excellent written and verbal communication skills</li>
</ul>

<h5 id="preferred-qualifications">Preferred qualifications</h5>

<ul>
  <li>Expertise in genomics</li>
  <li>Experience with pipeline or workflow automation</li>
  <li>Familiarity with software development best practices</li>
  <li>Experience configuring and deploying analyses on a cloud infrastructure</li>
  <li>Experience and willingness to participate in team decision-making processes</li>
</ul>

<p>The Fred Hutch is located in South Lake Union in Seattle, WA and offers a dynamic work environment with cutting-edge science and computational resources. The position is available immediately with flexible starting dates. Informal inquiries are welcome. Applications will be accepted until the position is filled. We offer a competitive salary commensurate with skills and experience, along with benefits. The Fred Hutch and the Bedford lab are committed to improving diversity in the computational sciences. Applicants of diverse backgrounds are particularly encouraged to apply. Depending on the applicant, this position could be a full-time salaried employee, a part-time employee, or a contracted consultant. An ideal candidate would be local to the Seattle area or willing to relocate, but remote work is also an option.</p>

<p>To apply for this position please go to the <a href="https://careers-fhcrc.icims.com/jobs/19821/job">Fred Hutch Careers Job ID 19821</a>.</p>

<p>To aid in applicant review, a coding sample is requested. We’re happy to review whatever you’re most proud of (in any programming language). If you don’t have code that can be publicly shared, that’s okay. Please apply anyway and just let us know that this isn’t available.</p>

<p><em>If you think you might be a great fit for this position but are concerned about meeting all qualifications, we’d like to hear from you. Please email Trevor Bedford at tbedford<span style="display:none">obfuscate</span>@fredhutch.org or John Huddleston at jhuddles<span style="display:none">obfuscate</span>@fredhutch.org.</em></p>

<hr />

<h3 id="full-stack-developer">Full-stack Developer</h3>

<p>Position for a full-stack developer is available immediately in the Bedford lab at the Fred Hutch to work on an open-source software platform for genomic epidemiology and evolutionary analysis of pathogens including SARS-CoV-2, influenza and Ebola virus. This position will contribute to ongoing work on Nextstrain, one of the lab’s major projects.</p>

<p><a href="https://nextstrain.org">Nextstrain</a> is an award-winning tool for tracking infectious disease epidemics developed in collaboration with the <a href="https://neherlab.org">Neher lab</a> at the University of Basel. Nextstrain won the <a href="https://www.nih.gov/news-events/news-releases/open-science-prize-announces-epidemic-tracking-tool-grand-prize-winner">Open Science Prize</a> in Feb 2017 and has been instrumental in analysis of the <a href="https://nextstrain.org/sars-cov-2">SARS-CoV-2 pandemic</a>, <a href="https://nextstrain.org/community/inrb-drc/ebola-nord-kivu">Ebola outbreaks</a>, <a href="https://nextstrain.org/zika/">Zika spread</a> in the Americas and is used by the World Health Organization to aid in the process of <a href="https://nextstrain.org/flu/">influenza vaccine strain selection</a>.</p>

<h4 id="responsibilities-1">Responsibilities</h4>

<p>This role would be responsible for development work up-and-down the entire Nextstrain software stack and involve both back-end and front-end development. All development occurs in an open-source fashion via <a href="https://github.com/nextstrain">github.com/nextstrain</a>. Specific priorities currently include infrastructure and pipelines to ingest and curate genomic data from public databases, optimizing use of cloud computing services to process this data, services to host and share analyses uploaded by Nextstrain users, and development of <a href="https://docs.nextstrain.org/projects/cli/en/latest/">command line tools</a> for working with Nextstrain. Informatic work focuses on development of the <a href="https://github.com/nextstrain/augur">Augur bioinformatics toolkit</a> and pathogen-specific workflows. Front-end work focuses on user functionality at nextstrain.org, including management of cloud computing and storage, as well as visualization improvements to the <a href="https://github.com/nextstrain/auspice">Auspice visualization JavaScript application</a>. Contributing to <a href="https://docs.nextstrain.org/en/latest/">documentation on the Nextstrain software stack</a> is a vital responsibility of this position.</p>

<p>Interfacing with project collaborators in-person and online is a key aspect of this position. The developer will work within a small team of existing members of the Bedford lab as well as other contributors to Nextstrain. The Nextstrain team communicates openly about project and organizational decisions and encourages participation by all team members in the decision-making process.</p>

<h4 id="qualifications-1">Qualifications</h4>

<h5 id="minimum-qualifications-1">Minimum qualifications</h5>

<ul>
  <li>Fluency in at least one high-level programming language, such as Python, R, Ruby, JavaScript or Perl</li>
  <li>Excellent written and verbal communication skills</li>
  <li>Experience in the following areas:
    <ul>
      <li>Web development</li>
      <li>Database systems</li>
      <li>Cloud infrastructure</li>
      <li>Software engineering and documentation best practices</li>
    </ul>
  </li>
</ul>

<h5 id="preferred-qualifications-1">Preferred qualifications</h5>

<ul>
  <li>Experience working with genomic data</li>
  <li>Systems integration</li>
  <li>Experience designing effective data visualizations</li>
  <li>Experience and willingness to participate in team decision-making processes</li>
</ul>

<p>The Fred Hutch is located in South Lake Union in Seattle, WA and offers a dynamic work environment with cutting-edge science and computational resources. The position is available immediately with flexible starting dates. Informal inquiries are welcome. Applications will be accepted until the position is filled. We offer a competitive salary commensurate with skills and experience, along with benefits. The Fred Hutch and the Bedford lab are committed to improving diversity in the computational sciences. Applicants of diverse backgrounds are particularly encouraged to apply. Depending on the applicant, this position could be a full-time salaried employee, a part-time employee, or a contracted consultant. An ideal candidate would be local to the Seattle area or willing to relocate, but remote work is also an option.</p>

<p>To apply for this position please go to the <a href="https://careers-fhcrc.icims.com/jobs/19820/job">Fred Hutch Careers Job ID 19820</a>.</p>

<p>To aid in applicant review, a coding sample is requested. We’re happy to review whatever you’re most proud of (in any programming language). If you don’t have code that can be publicly shared, that’s okay. Please apply anyway and just let us know that this isn’t available.</p>

<p><em>If you think you might be a great fit for this position but are concerned about meeting all qualifications, we’d like to hear from you. Please email Trevor Bedford at tbedford<span style="display:none">obfuscate</span>@fredhutch.org or John Huddleston at jhuddles<span style="display:none">obfuscate</span>@fredhutch.org.</em></p>

			</div>
		</div>
		<div class="col-lg-1"></div>
	</div>

	<div class="bigspacer"></div>
	<div class="spacer"></div>

</div>


	</div>

	<div id="footer"><span style="display:none">foo</span></div>

</body>
</html>
